src/express.ts

```diff
@@ -10,7 +10,10 @@ import { parseVariable } from './parse';
 import { StartSubscriptionEvent, SubscriptionManager } from './subscriptions';
 import { logger } from './logger';
 
-export type ErrorHandler = (res: express.Response, error: any) => void;
+export type ErrorHandler = (
+  res: express.Response,
+  error: ReadonlyArray<Error>
```

Comment:
```suggestion
  errors: ReadonlyArray<any>
```

I would use `any` because it might be GraphQLError or a different exception.